.github/workflows/publish.yml

```diff
@@ -41,19 +41,7 @@ jobs:
         run: mdbook build -d book
 
       - name: Zip exercise templates
-        run: cd book/exerciser && zip --recurse-paths ../html/comprehensive-rust-exercises.zip comprehensive-rust-exercises/
-
-      - name: Build all translations
-        run: |
-          for po_lang in ${{ env.LANGUAGES }}; do
-              echo "::group::Building $po_lang translation"
-              MDBOOK_BOOK__LANGUAGE=$po_lang \
-              MDBOOK_OUTPUT__HTML__SITE_URL=/comprehensive-rust/$po_lang/ \
-              mdbook build -d book/$po_lang
-              (cd book/$po_lang/exerciser && zip --recurse-paths ../html/comprehensive-rust-exercises.zip comprehensive-rust-exercises/)
-              mv book/$po_lang/html book/html/$po_lang
-              echo "::endgroup::"
-          done
+        run: cd book && find . -name "exerciser" -type d -exec bash -c 'zip --recurse-paths ./html/$(dirname {})/comprehensive-rust-exercises.zip {}/comprehensive-rust-exercises/' \;
```

Comment:
I don't think this is doing the same thing that the original logic was doing. It looks like the original was putting the archive at `book/$po_lang/html/comprehensive-rust-exercises.zip`, but with the version you have here they would end up at `book/html/$po_lang/comprehensive-rust-exercises.zip`. Is this change intentional?